runtime/transformations/components/index-operations/reset_index_and_save_as_column_100_77258f6a-db49-42a6-8133-7fcfd5bffddb.py

```diff
@@ -0,0 +1,141 @@
+"""Documentation for Reset Index and Save as Column
+
+# Reset Index and Save as Column
+## Description
+Uses the pandas reset_index method to reset the index of a DataFrame and save the previous index as a column.
+
+## Inputs
+* **df_to_reset_index** (Pandas DataFrame): The input DataFrame the index of which will be reset.
```

Comment:
ich glaube die Koncention ist überall _DataFrame_ statt _Pandas DataFrame_ zu schreiben in den Klammern. Bitte nochmal prüfen.

Reply:
Ich finde in allen anderen Komponenten auch "Pandas DataFrame"